doc/source/docs/user_guide/io.rst

```diff
@@ -225,6 +231,14 @@ by using the :meth:`geopandas.GeoDataFrame.to_postgis` method.
     countries_gdf.to_file("package.gpkg", layer='countries', driver="GPKG")
     cities_gdf.to_file("package.gpkg", layer='cities', driver="GPKG")
 
+**Writing with multiple geometry columns**::
+
+    countries_gdf["country_center"] = countries_gdf["geometry"].centroid
+    # countries_gdf.to_file("countries.geojson", driver='GeoJSON') # Fails because GeoJSON can't contain multiple geometry columns
+    countries_gdf["country_center"] = countries_gdf["country_center"].to_wkt()
+    countries_gdf.to_file("countries.geojson", driver='GeoJSON')
+
+For multi-layer formats such as geopackage, additional geometry columns can be written to separate layers.
```

Comment:
```suggestion
For multi-layer formats such as geopackage it is also possible to write additional geometry columns to separate layers, instead of saving as WKT or WKB within a single layer.
```